### ISSUE TITLE
Please upgrade sharp

### ISSUE THREAD

**Issue by grumpyshiba** [mahnunchik/gulp-responsive#155]
Please upgrade sharp version. It doesn't work on new MacOSX (doesn't compile as it uses broken libvps link)

**Comment by SamuelMiller** [comment]
Yes please update! I too am not able to this plugin on Mac OSX using Node v16.15.0 and Gulp (CLI version: 2.3.0
Local version: 4.0.2). It seems like a lot of Gulp plugins have been abandoned--no longer updated. Is Gulp considered abandonware, out of fashion? Having just learnt about Gulp and appreciating its utility, I am sad to see the plugin development for it wane.

**Comment by Northward-Design** [comment]
@grumpyshiba @SamuelMiller 
This fork worked for me [aidanmontare/gulp-responsive](https://github.com/AidanMontare/gulp-responsive)
Hopefully @mahnunchik will update this and [npm Readme](https://www.npmjs.com/package/gulp-responsive) at some point

**Comment by houmie** [comment]
@Northward-Design How can I use brew with the fork you mentioned?  I don't think that's possible, right? DO you have any suggestions how to install this?
